lib/executors/PreExecutor.js

```diff
@@ -220,6 +221,25 @@ define([
 					}
 				});
 
+				if (has('host-node')) {
+					var files, splice;
```

Comment:
One `var` per var, please (code style preference). It would also be better to define these further down, inside the anonymous function where they are used.